addon/components/power-select/options.ts

```diff
@@ -34,6 +34,11 @@ if(typeof FastBoot === 'undefined'){
 export default class Options extends Component<Args> {
   private isTouchDevice = isTouchDevice
   private hasMoved = false
+  private mouseOverHandler = (_: MouseEvent): void => {}
+  private mouseUpHandler = (_: MouseEvent): void => {}
+  private touchEndHandler = (_: TouchEvent): void => {}
+  private touchMoveHandler = (_: TouchEvent): void => {}
+  private touchStartHandler = (_: TouchEvent): void => {}
```

Comment:
This declaration seemed to make satisfy the TS compiler. Is this an appropriate way to declare a Function in Typescript that is meant to be overwritten at runtime.

Reply:
Looks correct to me.